src/components/MovieInfo.tsx

```diff
@@ -1,4 +1,6 @@
 import React from "react";
+import { BiCameraMovie } from "react-icons/bi";
+import { PiTelevisionSimple } from "react-icons/pi";
```

Comment:
Let's do this instead, for the instantiation of these components, shall we?
`import { BiCameraMovie, PiTelevisionSimple  } from "react-icons/bi";`

Reply:
I get an error when I do that import, since the module is different for each icon. One is react-icons/bi, while the other is react-icons/pi

Reply:
True, I misread that, sorry about the confusion, let's discard this change and I will resolve this conversation so that we no longer deal with this change.